jtransc-rt/resources/hx/N.hx

```diff
@@ -27,31 +26,95 @@ typedef JavaDouble = {% CLASS java.lang.Double %}
 // https://haxe.io/roundups/wwx/c++-magic/
 {{ HAXE_CLASS_ANNOTATIONS }}
 #if cpp
-@:headerClassCode('inline static int _i2b(int v) { return (char)v; }; inline static int _i2s(int v) { return (short)v; }; inline static int _i2c(int v) { return (unsigned short)v; };')
+@:headerClassCode('
+	inline static int _i2b(int v) { return (char)v; };
+	inline static int _i2s(int v) { return (short)v; };
+	inline static int _i2c(int v) { return (unsigned short)v; };
+	inline static void _memcpy(unsigned char *dst, int dstPos, unsigned char *src, int srcPos, int shift, int count) {
+		::memcpy(
+			&dst[dstPos << shift],
+			&src[srcPos << shift],
+			count << shift
+		);
+	};
+
+	inline static void _memset_N2(unsigned char *dst, int dstPos, int count, int value) {
```

Comment:
C++ generated from Haxe. I have tried `intNN_t` and they are not defined. Which is a bit, meh. Not sure that `long long int` resolves to `int64_t` in all targets.